### DISCUSSION TITLE
Performance problem

### DISCUSSION THREAD

**Discussion by FugaziScript** [Blinue/Magpie#22] [General]
The magpie uses a lot of resources. It is able to take 8% - 20% of the processor in the task manager. It also causes visible latency. In video upscaling it is not a problem, but in games (a lot of people are here for that) it’s useless.
It would be great, if you can do something about that problem. My processor is ryzen 7 4800u.
Also GDI seems to be working better with little to no latency.
![Zrzut ekranu 2021-08-02 142206](https://user-images.githubusercontent.com/82718856/127861195-ce4a2c53-975a-47bc-93a3-1b6d1e877481.png)

**Comment by Blinue** [comment]
I've posted an issue #23
